tools/install.sh

```diff
@@ -72,38 +78,21 @@ else
     exit 2;
 fi
 
-if [[ -n $(which composer) ]]; then
-    echo ""
-    echo "PHP Composer appears to be installed."
-    composer_scr="composer install --no-dev"
-elif [[ -x ../composer ]]; then
-    echo ""
-    echo "PHP Composer appears to be installed."
-    composer_scr="./composer install --no-dev"
-else
-    echo ""
-    echo "PHP Composer does not appear to be installed. Please install it before running this script."
-    echo ""
-    echo "(e.g. wget https://getcomposer.org/installer"
-    echo "php installer --install-dir=/usr/local/bin --filename=composer)"
-    echo "while having root permission)";
+PHP_VERSION=$(php -v | head -n 1 | cut -d " " -f 2 | cut -f1-2 -d".")
```

Comment:
This may be a weak way to test the PHP version given that the CLI version of PHP can differ from the version Apache is configured to run with.

Not blocking this PR but I'm going to create an issue so that someone creates a more robust test in the future.